packages/slate-hyperscript/rollup.config.js

```diff
@@ -0,0 +1,67 @@
+import resolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import babel from 'rollup-plugin-babel'
+import uglify from 'rollup-plugin-uglify'
+import sourcemaps from 'rollup-plugin-sourcemaps'
+import pkg from './package.json'
+
+// UMD build for browsers
+const umdConfig = {
+  input: 'src/index.js',
+  output: {
+    file: pkg.browser,
+    name: 'SlateHyperscript',
+    format: 'umd',
+    exports: 'named',
+    globals: {
+      slate: 'Slate',
+    },
+  },
+  external: [
+    'slate',
+  ],
+  plugins: [
+    resolve(),
+    commonjs({
+      exclude: ['src/**'],
+    }),
+    babel({
+      include: ['src/**']
+    }),
+  ]
+}
+
+// Additional UMD minified build based off of the unminified config
+const umdConfigMin = Object.assign({}, umdConfig)
+umdConfigMin.output = Object.assign({}, umdConfig.output, { file: pkg.browserMin })
+umdConfigMin.plugins = umdConfig.plugins.slice(0).concat(uglify())
+
+// CommonJS (for Node) and ES module (for bundlers) build.
+const moduleConfig = {
+  input: 'src/index.js',
+  external: [
+    'is-empty',
+    'is-plain-object',
+    'slate',
+    'slate-dev-logger',
```

Comment:
Why are these ones other than `slate` external? They aren't peer dependencies?

Reply:
In this context, `external` tells rollup that the package is an external npm dependency rather than something that it is supposed to bundle into your built output file.

Reply:
Is there really no way other than manually managing each dependency here? This seems like a big downside to using Rollup?

Reply:
Hmm, I can use [rollup-plugin-auto-external](https://www.npmjs.com/package/rollup-plugin-auto-external), that'll reduce this bit for the CJS/ES sections. It's still probably needed for the UMD builds though because it's not so cut-and-dry (we actually WANT to bundle in some of the smaller packages).